script/github-actions/get-changed-apps.js

```diff
@@ -26,16 +26,16 @@ const getManifests = filePath => {
 
 /**
  * Gets the sliced manifest(s) of a file's root app folder with some added properties. The
- * app's root folder must be on the given allow list, otherwise returns null.
+ * app's root folder must be on the given allow list, otherwise returns an empty array.
  *
  * @param {string} filePath - Relative file path.
  * @param {Object} allowedApps - List of allowed apps.
- * @returns {Object[]|null} Sliced manifests of allowed apps. Otherwise null.
+ * @returns {Object[]} Sliced manifests of allowed apps.
  */
 const getAllowedApps = (filePath, allowedApps) => {
   const appsDirectory = 'src/applications';
 
-  if (!filePath.startsWith(appsDirectory)) return null;
+  if (!filePath.startsWith(appsDirectory)) return [];
```

Comment:
Not a necessary change, but I feel like returning an array instead of `null` makes the function more predictable, especially when trying to parse it's output

Reply:
I agree. It's always better to have a single return type instead of multiple. Simpler and less error-prone.